src/movies.js

```diff
@@ -1,25 +1,95 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+/* const getAllDirectors = (moviesArray) => moviesArray.map(movie => movie.director); */
+
+function getAllDirectors(moviesArray) {
+  /* const directors = moviesArray.map(function (movie) {
+    return movie.director;
+  }); */
+  const directors = moviesArray.map((movie) => movie.director);
+  return directors;
```

Comment:
Esto esta perfect!!! te lo refactorizo un poco para que vayais hacia donde queremos que vayais!
```suggestion
  return moviesArray.map((movie) => movie.director);
```